UlKit/UlKit/CatalogViewController.swift

```diff
@@ -3,9 +3,280 @@
 
 import UIKit
 
+/// Описывает все товары, которые находятся в каталоге
 final class CatalogViewController: UIViewController {
+    // MARK: - Constants
+
+    enum Constants {
+        static let categoryWomen = "Женщинам"
+        static let categoryMen = "Мужчинам"
+        static let categoryChildren = "Детям"
+
+        static let blockBrands = "brands"
+        static let blockBags = "bags"
+        static let blockShoes = "shoes"
+
+        static let fontVerdanaBold = "Verdana-Bold"
+        static let titleCatalog = "Каталог"
+
+        static let shoesNameImage = "womanShoes"
+        static let sandalsNameImage = "sandalsWoman"
+        static let brendsNameImage = "brends"
+        static let bagNameImage = "bagWoman"
+        static let shoesWomanNamePicture = "shoesWoman"
+
+        static let manShoesNameImage = "manShoes"
+        static let manSneakersNameImage = "manSneakers"
+        static let shoesManNameImage = "shoesMan"
+        static let bagManNameImage = "bagMan"
+
+        static let kidsSneakersNameImage = "kidsSneakers"
+        static let babySneakersNameImage = "babySneakers"
+        static let childrensSneakersNameImage = "childrensSneakers"
+        static let babyBackpackNameImage = "babyBackpack"
+    }
+
+    // MARK: - Private Properties
+
+    private let categories = [Constants.categoryWomen, Constants.categoryMen, Constants.categoryChildren]
+
+    // MARK: - Visual Components
+
+    private lazy var categoriesSegmentedControl = UISegmentedControl(items: self.categories)
+
+    private lazy var shadowShoesView = createShadowView()
+
+    private lazy var shadowSneakersView = createShadowView()
+
+    private lazy var shoesImageView: UIImageView = createImageView(nameImage: Constants.shoesNameImage)
```

Comment:
тут если отказаться от лишних строк с именами картинок, можно было бы передавать как параметр сразу картинку
```suggestion
    private lazy var shoesImageView: UIImageView = createImageView(image: .womanShoes)
```